layouts/partner/aws.html

```diff
@@ -7,8 +7,15 @@ <h1>{{ .Params.hero.title }}</h1>
             <div class="w-full order-last mt-8 lg:mt-0 lg:order-first lg:w-1/2 px-5">
                 <p class="mr-4">{{ .Params.hero.description }}</p>
                 <div class="flex flex-col lg:flex-row mt-5">
-                    <a class="btn btn-lg text-center mb-4 lg:mb-0 lg:mr-4" href="{{ relref . "/docs/get-started/aws" }}">Get Started</a>
-                    <a class="btn btn-lg btn-orange-translucent text-center" href="{{ relref . "/resources#upcoming" }}">Join a Workshop</a>
+                    <div class="flex items-start mb-4 lg:mb-0 lg:mr-4">
+                        <a class="btn btn-lg text-center" href="{{ relref . "/docs/get-started/aws" }}">Get Started</a>
+                    </div>
+                    <div class="flex items-start">
+                        <a class="btn btn-lg btn-orange-translucent text-center" href="{{ relref . "/resources#upcoming" }}">Watch a Workshop</a>
+                    </div>
+                    <div class="flex-grow flex mt-4 lg:mt-0 lg:justify-center">
```

Comment:
```suggestion
                    <div class="flex-grow mt-4 lg:mt-0 lg:justify-center">
```

Reply:
☝️ I don't think any of these are benefiting from having `display: flex` themselves, since they all only contain one item.

Reply:
👍